packages/widget-chat/src/containers/message-composer/index.js

```diff
@@ -9,18 +9,14 @@ import AddFileButton from '../../components/add-file-button';
 
 import styles from './styles.css';
 
-export class MessageComposer extends Component {
+// eslint-disable-reason Redux connect gives us shouldComponentUpdate smarts
+export class MessageComposer extends Component { // eslint-disable-line react/require-optimization
   constructor(props) {
     super(props);
     this.handleChange = this.handleChange.bind(this);
     this.handleKeyDown = this.handleKeyDown.bind(this);
   }
 
-  shouldComponentUpdate(nextProps) {
-    const props = this.props;
-    return props.value !== nextProps.value;
```

Comment:
Here I believe any changes in `conversation` store that goes through a reducer would cause a re-render.